loops_lab.js

```diff
@@ -0,0 +1,109 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+// let num = 24;
+
+while (num-- && num >= 2) {
+  console.log(num);
+}
+
+for (num >= 2; num-= 1;) {
+  console.log(num);
+}
+
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+let max = 32;
+
+// while (max <= 33) {
+//   if (max % 2 === 0) {
+//     console.log(max + " is even.");
+//   } else {
+//     console.log(max + " is odd.");
+//   }
+//   max++;
+// }
+
+for (let max = 0; max <= 32;) {
+  if (max % 2 === 0) {
+    console.log(max + " is even.");
+  } else {
+    console.log(max + " is odd.");
+  }
+  max ++;
+}
+
+// 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+// For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18").
+//   _Bonus_ think of another way to solve it.
+//     <details>
+//       <summary>
+//         Hint
+//       </summary>
+//       Find the final number and increment the loop by 9.
+//     </details>
+//
+
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, exclusive.
+let myNumber = 1;
+while (myNumber < 100) {
+  if (myNumber % 5 === 0 && myNumber > 5 && myNumber % 2 != 0) {
```

Comment:
This condition certainly works. Another way to solve this is to involve modulo 5 and !modulo 10.